appointment/lambda_function.py

```diff
@@ -0,0 +1,123 @@
+import json
+import time
+import datetime
+import re
+from db_wrapper import execute_statement, extract_records
+
+#Created by Nishant Acharya
+#Last Updated 4/22/2020
+
+def lambda_handler(event, context):
+    
+    #Getting the values to put into the row
+    student_id = int(event['student_id'])
+    supporter_id = int(event['supporter_id'])
+    appt_date = event['appt_date']
+    start_time = event['start_time']
+    duration_min = int(event['duration'])
+    type = int(event['type'])
+    cancelled = bool(event['cancelled'])
+    rating = int(event['rating'])
+    recommended = bool(event['recommended'])
```

Comment:
With lambda proxy integration the event is mapped as:
![image](https://user-images.githubusercontent.com/48414002/80043402-928c8200-84cf-11ea-807b-61e2ca43cfb6.png)
The request body would be in "body", to extract the student id I believe it would be something like:
              student_id = json.loads(event["body"])["student_id"]

Reply:
Updated the code to reflect the above mentioned changes